openshift/kustomize/tekton/base/pipelines/buildah-all.yaml

```diff
@@ -14,11 +14,15 @@ metadata:
     managed-by: kustomize
 spec:
   params:
-    - name: GIT_REF
+    - name: GIT_SOURCE
```

Comment:
There was a template bug between GIT_SOURCE and GIT_REF.  I have resolved this issue and now both the `master` and `dev` branch should work and trigger the pipeline.